ploting.R

```diff
@@ -0,0 +1,53 @@
+plotcovid <- function (data, 
+                       scale = c("s", "ls", "ps"), 
+                       by = c("all", "ha","age"), 
+                       time = c("all", "recent")) {
```

Comment:
should probably document these, or even better make the options more descriptive.
So, rather than `scale = c("s", "ls", "ps")`, you would use `scale_y = c("none", "log_scale", "per_capita")` and rather than `by = ...` you could say `subgroups = c("all", "health_authority", "age")`